architecture/infrastructure_components/web_console.adoc

```diff
@@ -49,6 +51,20 @@ link:../../install_config/master_node_configuration.html#master-configuration-fi
 configuration file parameter `corsAllowedOrigins`].
 endif::[]
 
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+[[web-console-cli-downloads]]
+
+== CLI Downloads
+You can download and unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console] for use on Linux, MacOSX, and Windows clients
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
+endif::[]
+:
+
+image::about_page.png["About Page"]
+
```

Comment:
You can probably just yank it out here. https://github.com/openshift/openshift-docs/pull/1608 is the PR for this, and it will get picked for 3.2 at the appropriate time per its milestone.